designcompose/src/main/java/com/android/designcompose/FrameRender.kt

```diff
@@ -477,7 +479,304 @@ internal fun ContentDrawScope.render(
         drawContent()
     }
 
-    if (useBlendMode) {
+    if (useBlendMode || opacity < 1.0f) {
```

Comment:
Why is the `opacity < 1.0f` added here? If this is added, doesn't a `canvas.save()` need to be called somewhere earlier also?

Reply:
I removed this, I applied it in error -- we handle `opacity < 1.0f` via `graphicsLayer` in non-squoosh, so it was a mistake to add this condition (and it was indeed causing the 1px separator test to crash).